Core/src/org/sleuthkit/autopsy/casemodule/services/TagNameDefinition.java

```diff
@@ -76,8 +98,20 @@
         this.knownStatus = status;
     }
 
+    static Collection<TagNameDefinition> getStandardTagNameDefinitions() {
+        return STANDARD_TAGS_DEFINITIONS.values();
+    }
+
+    static Collection<TagNameDefinition> getProjectVICDefaultDefinitions() {
+        return PROJECT_VIC_TAG_DEFINITIONS.values();
+    }
```

Comment:
It would be better to not hand out a reference to the internal data structure. The TagName objects are immutable, but the  collections are not.